tapiro/core/src/test/scala/io/buildo/tapiro/TapiroSuite.scala

```diff
@@ -5,7 +5,7 @@ import java.nio.file.Files
 class TapiroSuite extends munit.FunSuite {
 
   check(
-    "http4s",
+    "tapir and http4s endpoints",
```

Comment:
minor: let's try to keep test names short and without spaces, so that they're easily selectable for `testOnly`.

E.g.

```scala
tapiroCore/testOnly -- io.buildo.tapiro.TapiroSuite.tapir-http4s-endpoints
```